crates/eframe/src/epi.rs

```diff
@@ -369,6 +369,10 @@ pub struct NativeOptions {
     /// The folder where `eframe` will store the app state. If not set, eframe will get the paths
     /// from [directories_next].
     pub persistence_path: Option<std::path::PathBuf>,
+
+    /// Controls whether colors should be dithered to minimize banding.
+    /// Defaults to true.
+    pub dithering: bool,
```

Comment:
I have a bad feeling about making this opt-out rather than opt-in. In theory textures that are already in srgb8 should be fine since the dithering shouldn't affect those (see the `scale down the noise slightly to ensure flat colors aren't getting dithered` bit in the shader(s)). But let's stick with it.

In any case this is one of the many things that should be clarified here. Let's try:

```suggestion
    /// Controls whether to apply dithering to minimize banding artifacts.
    /// 
    /// Dithering assumes an sRGB output and thus will apply noise to any input value that lies between
    /// two 8bit values after applying the sRGB OETF function, i.e. if it's not a whole 8bit value in "gamma space".
    /// This means that only inputs from texture interpolation and vertex colors should be affected in practice.
    ///
    /// Defaults to true.
    pub dithering: bool,
```

I thiiiink this is accurate without being two confusing. Counter-feedback welcome.